x-pack/plugins/observability_solution/profiling/public/utils/formatters/as_integer.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function asInteger(value: number) {
+  return Math.round(value ?? 0).toString();
```

Comment:
Do you mind adding unit tests for this, please?

Reply:
Added tests